### ISSUE TITLE
Support more sorting type

### ISSUE THREAD

**Issue by issadarkthing** [issadarkthing/gomu#45]
As of recent merged pr #41 where I mention we cannot directly access the creation time of file https://github.com/issadarkthing/gomu/pull/41#issuecomment-800277029. I am still indecisive on we should use `ctime` or `mtime` as both have different behavior.

> mtime changes if tag is changed but does not change if renamed
ctime changes if audio file is renamed but does not change if tag is changed

This leads me to add more sorting option to leave the decision up to the user. Few sort options I would like to implement are ctime, name and audio length.

Let me know which sorting options are redundant and irrelevant to the app itself. More sorting options suggestion are welcomed!

**Comment by tramhao** [comment]
In my scenario, perhaps sorting by Tags could be an option too, like artist. Album nowadays are not so effective as most songs are singles. 
And about the hot keys, could we do something like https://github.com/And678/goPlayer/blob/assets/screenshot.png, I mean showing some important keys as hint to users.

**Comment by issadarkthing** [comment]
> In my scenario, perhaps sorting by Tags could be an option too, like artist. Album nowadays are not so effective as most songs are singles.

So the sorting options will be added are ctime, name, audio length and audio name tag.

> And about the hot keys, could we do something like https://github.com/And678/goPlayer/blob/assets/screenshot.png, I mean showing some important keys as hint to users.

I can see this will be helpful for the beginner but I personally don't like it cluttering the screen. If we were to add this, maybe make it toggle-able in config file like `show_hint = false`.

**Comment by tramhao** [comment]
> I can see this will be helpful for the beginner but I personally don't like it cluttering the screen. If we were to add this, maybe make it toggle-able in config file like `show_hint = false`.

I agree with you. After a second thought, I think this is not very important as we already have very good help menu. And most keys are vim keys.
